webpack/stripes-module-parser.js

```diff
@@ -170,6 +170,15 @@ function parseAllModules(enabledModules, context, aliases) {
     if (moduleParser.warnings.length) {
       warnings = warnings.concat(moduleParser.warnings);
     }
+
+    // Interim measure to allow ui-notes to offer both an app and a plugin
+    // until STRIPES-148 is complete. See STRIPES-357
```

Comment:
You should update the JIRAs in the comments.  I assume you mean STCOR-148 and STCOR-357 here.

Reply:
True!